### PR TITLE
Add recent transactions visual

### PR DESCRIPTION
**Pull request by Spedwards**

### CODE REVIEW

**Comment by Spedwards** [comment]
I can assure you that sold transactions work fine. I need someone to test to see if deductions display correctly when a purchase is made.

File: visuals.js [hunk 1]
```diff
@@ -196,6 +199,40 @@ module.exports = class Visuals {
 		}
 	}
 	
+	static drawTransactions(room) {
+		if (room.terminal) {
+			const vis = new RoomVisual(room.name);
+			const x = room.terminal.pos.x;
+			let y = room.terminal.pos.y - 1;
+			
+			const transactions = _([...Game.market.incomingTransactions, ...Game.market.outgoingTransactions])
```
Comment (karlthepagan) [inline visuals.js]: change to _(incoming).concat(outgoing)

Comment (Spedwards) [inline visuals.js]: There should be absolutely no difference, but consider it done.

File: visuals.js [hunk 2]
```diff
@@ -196,6 +199,39 @@ module.exports = class Visuals {
...
+			
+			if (transactions.length === 2) {
+				y -= 0.4;
+			}
+			
+			transactions.forEach(transaction => {
+				const outgoing = transaction.sender.username === room.controller.owner.username;
+				const colour = outgoing ? '#00FF00' : '#FF0000';
+				const prefix = outgoing ? '+' : '-';
+				const text = `${prefix}${transaction.amount * transaction.order.price}`;
```
Comment (OverlordQ) [inline visuals.js]: Not all transactions have an order, so this will fail.

**Pull request merged**